packages/aws-cdk-lib/aws-rds/lib/serverless-cluster.ts

```diff
@@ -445,19 +458,34 @@ abstract class ServerlessClusterNew extends ServerlessClusterBase {
     const minCapacity = options.minCapacity;
     const maxCapacity = options.maxCapacity;
 
+    const timeoutAction = options.timeoutAction;
+
     if (minCapacity && maxCapacity && minCapacity > maxCapacity) {
       throw new Error('maximum capacity must be greater than or equal to minimum capacity.');
     }
 
-    const secondsToAutoPause = options.autoPause?.toSeconds();
-    if (secondsToAutoPause && (secondsToAutoPause < 300 || secondsToAutoPause > 86400)) {
-      throw new Error('auto pause time must be between 5 minutes and 1 day.');
-    }
+    const secondsBeforeTimeout = options.secondsBeforeTimeout.toSeconds();
```

Comment:
```suggestion
    const secondsBeforeTimeout = options.secondsBeforeTimeout?.toSeconds();
```

This is causing the build to fail.